lang/cs/Org.Apache.REEF.IO.Tests/TestLocalFileSystem.cs

```diff
@@ -136,6 +136,29 @@ public void TestCopy()
             Assert.False(fs.Exists(sourceUri));
         }
 
+        [Fact]
+        public void TestIsDirectory()
+        {
+            var fs = GetFileSystem();
+            // Directory check
+            var directoryUri = new Uri(Path.Combine(Path.GetTempPath(), TempFileName) + "/");
+            fs.CreateDirectory(directoryUri);
+            Assert.True(fs.IsDirectory(directoryUri));
```

Comment:
add an assert message, e.g.
```C#
Assert.True(fs.IsDirectory(directoryUri), ".IsDirectory() failed on: " + directoryUri);
```

Reply:
fixed

---
In reply to: [198678451](https://github.com/apache/reef/pull/1473#discussion_r198678451) [](ancestors = 198678451)